### DISCUSSION TITLE
Hudl 1 problem.

### DISCUSSION THREAD

**Discussion by AdrianCollins10** [remay/tmf-hudl#46] [General]
I have recently aquired a hudl 1 from a friend and wanted to let my granddaughter have it so I did a full reset and found that it got stuck on the tesco server problem. I have installed your tmf firmware but now it wont access youtube as it wants to update it. I have added my playstore account but it fails to access playstore saying there is a playstore server error. I have googled this but all it says is to reset cache but that doesn't work. Is there anything else I can try?

**Comment by richardbulger** [comment]
Recently dusted on a Hudl 1 too. Installed custom ROM fine, signed in to google account, but play store services not updating

**Comment by remay** [comment]
Sorry for the delay to my response.   Unfortunately Google Play Store no longer supports Android earlier than v5.0, and the Hudl 1 is v4.2.2.

See https://github.com/remay/tmf-hudl/issues/43 for more information and a potential way forwards.
